react-native-reanimated.d.ts

```diff
@@ -328,6 +328,16 @@ declare module 'react-native-reanimated' {
       value: Adaptable<number>,
       config: InterpolationConfig,
     ): AnimatedNode<number>;
+    export function interpolateColors(
+      animationValue: Adaptable<number>,
+      {
+        inputRange,
+        outputColorRange
+      }: {
+        inputRange: ReadonlyArray<Adaptable<number>>;
```

Comment:
After adding string handling this shouldn't be `ReadonlyArray<Adaptable<number | string>>;`?